scripts/migrate-live-find-references-step-4.bash

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-set -e
```

Comment:
this will now continue on any errors, such as if `grep` doesn't find any matches